DuckDuckGo/FirewallManager.swift

```diff
@@ -95,6 +95,10 @@ class FirewallManager: FirewallManaging {
                 statusDidChange()
             }
         } catch {
+            #if !targetEnvironment(simulator)
+                Pixel.fire(pixel: .appTPFailedToAccessPreferences, error: error)
+            #endif
```

Comment:
This is to avoid sending useless pixels from the simulator, as these can never run in the sim